packages/core/src/js/feedback/utils.ts

```diff
@@ -14,3 +21,29 @@ export const isValidEmail = (email: string): boolean => {
   const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
   return emailRegex.test(email);
 };
+
+/**
+ * Converts base64 string to Uint8Array on the web
+ * @param base64 base64 string
+ * @returns Uint8Array data
+ */
+export const base64ToUint8Array = (base64: string): Uint8Array => {
+  if (typeof atob !== 'function' || !isWeb()) {
+    throw new Error('atob is not available in this environment.');
+  }
+
+  const binaryString = atob(base64);
+  return new Uint8Array([...binaryString].map(char => char.charCodeAt(0)));
+};
+
+export const feedbackAlertDialog = (title: string, message: string): void => {
+  /* eslint-disable @typescript-eslint/ban-ts-comment, no-restricted-globals, no-alert, @typescript-eslint/no-unsafe-member-access */
+  // @ts-ignore
```

Comment:
I used `ts-ignore` to avoid touching the global state and compilerOptions to use `window`. I'll be happy to iterate if there is better suggestion for this 🙇

Reply:
One thing you could try is altering the worldwide.ts file at packages/core/src/js/utils/worldwride.ts

Change the ReactNativeInternalGlobal and add the following parameter:
`  alert?: (message: string) => void;`

So with that you could use the following code

```typescript
  if (isWeb() && typeof RN_GLOBAL_OBJ.alert !== 'undefined') {
    RN_GLOBAL_OBJ.alert(`${title}\n${message}`);
  } else {
```

Reply:
Thank you for the suggestion @lucas-zimerman 🙇 
This actually seems to work pretty well and there is no need to suppress any typescript checks.
Updated with 28738bf40e7d3e82c2549f59d1e93120e837e230